src/components/Stepper/index.js

```diff
@@ -204,6 +183,16 @@ const Stepper = ({
   const isMinusDisabled = inputValue <= minValue || disabled
   const isPlusDisabled = inputValue >= maxValue || disabled
 
+  const centerTouchableRef = useRef(null)
+
+  const [inputMaxWidth, setInputMaxWidth] = useState()
+
+  useEffect(() => {
```

Comment:
This was to correct which behaviour ?

Reply:
Input width but I corrected it with another way thanks to @DorianMaliszewski